src/utils/ExtensionUtils.js

```diff
@@ -166,21 +166,12 @@ define(function (require, exports, module) {
         var url     = PathUtils.isAbsoluteUrl(path) ? path : getModuleUrl(module, path);
         let result = new $.Deferred();
 
-        if(module.uri && module.uri.startsWith('/fs/app/extensions/')){
```

Comment:
removed file-based loading as extensions are always loaded from either the virtual server or phoenix src url